DataManagementSystem/scripts/dirac-dms-create-replication-request.py

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env python
-""" Create a DIRAC transfer/replicateAndRegister request to be executed
-    by the DMS Transfer Agent
-"""
+"""Create a DIRAC transfer/replicateAndRegister request to be executed by the RequestExecutingAgent."""
```

Comment:
This script is a duplicate in functionality to dirac-dms-replicate-and-register-request, one of which could be deprecated...

Reply:
Please go ahead and remove one of them.

Reply:
No deprecation warning?

I would remove the dirac-dms-create-replication-request.

dirac-dms-replicate-and-register-request allows one to specify multiple targetSEs, instead of just one

Reply:
No need... go ahead.

Reply:
removed it